### PR TITLE
Oracle plugin docs

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

File: website/source/api/secret/databases/oracle.html.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- `max_connection_lifetime` `(string: "0s")` - Specifies the maximum amount of
+  time a connection may be reused. If <= 0s connections are reused forever.
+
+### Sample Payload
+
+```json
+{
+  "plugin_name": "oracle-database-plugin",
+  "allowed_roles": "readonly",
+  "connection_url": "system/Oracle@localhost:1521/OraDoc.localhost"
```
Comment (calvn) [inline website/source/api/secret/databases/oracle.html.md]: Missing comma

File: website/source/api/secret/databases/oracle.html.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  "max_open_connections": 5,
```
Comment (calvn) [inline website/source/api/secret/databases/oracle.html.md]: Missing comma

File: website/source/api/secret/databases/oracle.html.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  "max_connection_lifetime": "5s",
```
Comment (calvn) [inline website/source/api/secret/databases/oracle.html.md]: Extra comma

File: website/source/api/secret/databases/oracle.html.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## Configure Connection
+
+In addition to the parameters defined by the [Database
+Backend](/api/secret/databases/index.html#configure-connection), this plugin
+has a number of parameters to further configure a connection.
+
+| Method   | Path                         | Produces               |
+| :------- | :--------------------------- | :--------------------- |
+| `POST`   | `/database/config/:name`     | `204 (empty body)` |
```
Comment (calvn) [inline website/source/api/secret/databases/oracle.html.md]: The formatting seems off at the end

Comment (briankassouf) [inline website/source/api/secret/databases/oracle.html.md]: That's ok, the markdown table will fix the width

Comment (calvn) [inline website/source/api/secret/databases/oracle.html.md]: Yea, I was hoping the extra spaces to be added for better eyecandy ;)

File: website/source/docs/secrets/databases/oracle.html.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+# Oracle Database Plugin
+
+Name: `oracle-database-plugin`
+
+The Oracle Database Plugin is an external plugin for the Database
+backend. This plugin generates database credentials dynamically based on
+configured roles for the Oracle database.
+
+The Oracle Database Plugin does not live in the core vault code and can be found
```
Comment (jefferai) [inline website/source/docs/secrets/databases/oracle.html.md]: s/vault/Vault, s/code/code tree, s/it's/its, s/github/GitHub

Comment (jefferai) [inline website/source/docs/secrets/databases/oracle.html.md]: (or, s/github/git)

**Pull request merged**